macros/create_pipe.sql

```diff
@@ -13,13 +13,23 @@
 {%- set file_type = pipe.file_type %}
 
 {# set some defaults #}
-{%- set format_type_options = {
-    'skip_header': 1,
-    'null_if': ('', 'null'),
-  }
-  if file_type == 'CSV'
-  else {}
-%}
+{% if not pipe.match_by_column_name and not pipe.parse_headers %}
```

Comment:
seems like this is just removing the `skip_header` format option, which can be overwritten by adding any extra_format_options to the pipe's yaml. this should produce same results:
```yml
extra_format_options:
  skip_header: 0
```

Reply:
Ahhh, good call. Didn't realize it would overwrite the default.